source/includes/api/_fields.md

```diff
@@ -48,51 +44,412 @@ curl -X GET https://api.resourcewatch.org/v1/fields/1170030b-9725-4bfe-8fb4-1b0e
+                {
+                    "dataType": {
+                        "precision": "INT",
+                        "range": {
+                            "max": 2147483647,
+                            "min": -2147483648
+                        }
+                    },
+                    "grid": {
+                        "affineTransform": {
+                            "scaleX": 0.00833333376795053,
+                            "scaleY": -0.00833333376795053,
+                            "translateX": -180.0000000000001,
+                            "translateY": 90.00000782310963
+                        },
+                        "crsCode": "EPSG:4326",
+                        "dimensions": {
+                            "height": 18000,
+                            "width": 43200
+                        }
+                    },
+                    "id": "b1",
+                    "pyramidingPolicy": "MEAN"
+                }
+            ],
+            "geometry": {
+                "coordinates": [
+                    [
+                        [
+                            "-Infinity",
+                            "-Infinity"
+                        ],
+                        [
+                            "Infinity",
+                            "-Infinity"
+                        ],
+                        [
+                            "Infinity",
+                            "Infinity"
+                        ],
+                        [
+                            "-Infinity",
+                            "Infinity"
+                        ],
+                        [
+                            "-Infinity",
+                            "-Infinity"
+                        ]
+                    ]
+                ],
+                "type": "Polygon"
             },
-            {
-              "reason": "",
-              "$t": "2"
+            "id": "users/resourcewatch/cli_030_global_aridity",
+            "name": "projects/earthengine-legacy/assets/users/resourcewatch/cli_030_global_aridity",
+            "properties": {
+                "title": "Image:cli_030 _global_aridity"
+            },
+            "sizeBytes": "413389345",
+            "title": "Image:cli_030 _global_aridity",
+            "type": "IMAGE",
+            "updateTime": "2017-10-06T18:58:39.646890Z"
+        },
+        "tableName": "users/resourcewatch/cli_030_global_aridity"
+    }
+}
+```
+
+Google Earth Engine (GEE) is perhaps the biggest outlier when it comes to fields data. Because it relies on raster data, rather than tabular data, the data structure you'll find in a GEE dataset is very different, as you can see in the example on the side.
+
+The response consists of loading the details for a single image, with the most meaningful information being the `properties` (of an image) and `bands` (of said image). Those will be the fields you'll be able to retrieve when using the query endpoints.
+
+### NEX-GDDP
```

Comment:
Can we erease this bit we  no longer support  nexgddp/loca/rasdaman types

Reply:
since this is not yet scheduled to be rolled out, let's keep it in, and remove it later